src/main/java/com/google/sps/servlets/RetrieveActivitiesServlet.java

```diff
@@ -11,6 +11,8 @@
 import com.google.sps.data.Activity;
 import com.google.sps.data.Activity.Category;
 import static com.google.sps.data.ActivitiesUtility.getActivities;
+import static com.google.sps.data.GneralUtility.getParameter;
+
```

Comment:
remove extra line.

Reply:
In the same servlet file, there are extra lines at lines 32,33,34. Make sure to remove the two extra lines.

Reply:
This servlet is acutally no longer necessary. I'll go ahead and delete it.